docs/README.template.md

```diff
@@ -1,11 +1,11 @@
 <h1 align="center">
   <a href="https://interledger.org"><img src="ilp_logo.png" width="150"></a>
   <br>
-  ILP Client
+  ILP
 </h1>
 
 <h4 align="center">
-A low-level JS <a href="https://interledger.org">Interledger</a> sender/receiver library
+A low-level JS <a href="https://interledger.org">Interledger</a> library
```

Comment:
It might be more accurate to say "The Javascript client library for Interledger"